crates/cli/src/util.rs

```diff
@@ -361,7 +361,7 @@ Has the server rotated its keys?
 Remove the outdated identity with:
 \tspacetime identity remove {identity}
 Generate a new identity with:
-\tspacetime identity new --no-email --server {server}"
+\tspacetime identity new --no-email --server {server} --default"
```

Comment:
We should fix the UX issue in general, but regardless this is a good change to the warning to the user.